matchers/be_numerically_matcher.go

```diff
@@ -13,11 +13,25 @@ type BeNumericallyMatcher struct {
 }
 
 func (matcher *BeNumericallyMatcher) FailureMessage(actual interface{}) (message string) {
-	return format.Message(actual, fmt.Sprintf("to be %s", matcher.Comparator), matcher.CompareTo[0])
+	return matcher.FormatFailureMessage(actual, true)
 }
 
 func (matcher *BeNumericallyMatcher) NegatedFailureMessage(actual interface{}) (message string) {
-	return format.Message(actual, fmt.Sprintf("not to be %s", matcher.Comparator), matcher.CompareTo[0])
+	return matcher.FormatFailureMessage(actual, false)
+}
+
+func (matcher *BeNumericallyMatcher) FormatFailureMessage(actual interface{}, positive bool) (message string) {
```

Comment:
Could we maybe change `positive` here to `negated`, and flip the boolean expressions to be consistent with the Method names. Also, can we make this private `formatFailureMessage`?